src/RequestFeedback.cc

```diff
@@ -117,9 +117,8 @@ std::string SolverRequester::Feedback::asUserString( const SolverRequester::Opti
 
     case UPD_CANDIDATE_CHANGES_VENDOR:
     {
-      PoolItem highest = ui::asSelectable()(_objinst)->highestAvailableVersionObj();
       std::ostringstream cmdhint;
-      cmdhint << "zypper install " << highest.asString();
+      cmdhint << "zypper install --allow-vendor-change " << _objinst.name();
```

Comment:
I would not change the hint here. `"zypper install " << highest.asString()` gives you exactly the reported package. No opt for the solver to change it.

Reply:
Done